requirements.sh

```diff
@@ -22,7 +22,7 @@ os_name=$(grep -o '^NAME=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')
 os_version=$(grep -o '^VERSION_ID=.*' /etc/os-release | cut -f2 -d\" | sed 's/"//g')
 echo "Platform is ${os_name}, Version: ${os_version}"
 
-if [[  $os_name == *"Red Hat"* || $os_name == *"CentOS"*  ]]; then
+if [[ "$os_name" == *"Red Hat"* ]] || [[ "$os_name" == *"CentOS"* ]]; then
```

Comment:
Should this also be made sh compatible?

Reply:
Converted to a case statement. Now it is working.